.github/workflows/bump-version.yml

```diff
@@ -17,26 +17,26 @@ jobs:
         id: regex-match
         with:
           text: ${{ github.event.inputs.version }}
-          regex: '^(\d+.\d+).\d+(?:-(?:(beta\d+)|(pre)))?$'
```

Comment:
I assume the regex change is because the number is now optional?

Reply:
yep and because there's no beta anymore.